platform_umbrella/apps/control_server_web/lib/control_server_web/components/projects/batteries_form.ex

```diff
@@ -15,12 +15,16 @@ defmodule ControlServerWeb.Projects.BatteriesForm do
 
   def mount(socket) do
     groups = Catalog.groups_for_projects()
+    core_battery = KubeServices.SystemState.SummaryBatteries.core_battery()
+    # core_battery / config / usage will only be nil during some tests :grimacing:
```

Comment:
How is that possible? It seems so fundamental. Can you add how that happens as a comment until it gets cleaned up.

Reply:
It was a nightmare but I was able to fix the tests so that we don't this anymore.